src/lib/plugins/log.js

```diff
@@ -25,8 +44,9 @@ module.exports.server = function (serv, settings) {
   const logFile = path.join('logs', timeStarted + '.log')
 
   serv.log = message => {
-    readline.cursorTo(process.stdout, 0)
+    readline?.cursorTo(process.stdout, 0)
     message = moment().format('MMMM Do YYYY, HH:mm:ss') + ' ' + message
+    if (!isInNode) message = '[server] ' + message
```

Comment:
I think it always makes sense to do this when flying-squid is used as a dependency since there are other parts of an application that can do logging.

Reply:
why is disabling date display related with being in node or not ? what about making this an option instead ?

Reply:
I added `serv.formatMessage` so I can be flexible on my side